src/main/java/com/flightstats/hub/app/HubBindings.java

```diff
@@ -15,13 +15,7 @@
 import com.flightstats.hub.metrics.PeriodicMetricEmitter;
 import com.flightstats.hub.replication.ReplicationManager;
 import com.flightstats.hub.rest.*;
-import com.flightstats.hub.spoke.FileSpokeStore;
-import com.flightstats.hub.spoke.GCRunner;
-import com.flightstats.hub.spoke.SpokeClusterRegister;
-import com.flightstats.hub.spoke.SpokeFinalCheck;
-import com.flightstats.hub.spoke.SpokeReadContentDao;
-import com.flightstats.hub.spoke.SpokeStore;
-import com.flightstats.hub.spoke.SpokeWriteContentDao;
+import com.flightstats.hub.spoke.*;
```

Comment:
The one non-default IntelliJ setting we have (in the hub code, anyway) is that we don't splat our imports (I have my limit set to 999 or something silly). I think I may have neglected to specify that the last time you asked about how we format our code. Sorry!

Reply:
OOPs! I do not like these imports at all. i will replace them .